karma.conf.js

```diff
@@ -13,7 +13,7 @@ export default config => {
   config.set({
     browsers: ['PhantomJS'],
     singleRun: !ENV.isDevelopment(),
-    reporters: ['mocha'],
+    reporters: [ENV.isDevelopment() ? 'mocha' : 'dots'],
```

Comment:
Use the `mocha` reporter when in development (running local) and `dots` otherwise.